openlibrary/plugins/upstream/utils.py

```diff
@@ -1158,6 +1159,66 @@ def reformat_html(html_str: str, max_length: int | None = None) -> str:
         return ''.join(content).strip().replace('\n', '<br>')
 
 
+def get_isbn_10_and_13(isbns: str | list[str]) -> tuple[list[str], list[str]]:
```

Comment:
It may be more appropriate for this isbn related code to go in utils/isbn.py (as there are several methods like it there):

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/utils/isbn.py

Reply:
I moved this as suggested.